src/lib/ventilation.f90

```diff
@@ -726,7 +791,8 @@ subroutine estimate_flow(dpmus,dt,err_est)
     enddo !nunit
 
     ! the estimate of error for the iterative solution
-    err_est=err_est/(flow_sum*DBLE(num_units))
+    if(abs(flow_sum*dble(num_units)).gt.zero_tol) &
+         err_est = err_est/(flow_sum*dble(num_units))
```

Comment:
I get this is avoiding a divide by zero error  - is it the right thing to leave as is if sum of all flows is essentially zero?